### PR TITLE
Feature metno specific additions

### PR DESCRIPTION
**Pull request by TAlonglong**
Add some metno specific addtions

Add possibility to partly control the plot options for optional plots

Some flake8 adjustments
 - [x] Tests added 
 - [x] Tests passed 
 - [x] Passes ``git diff origin/master **/*py | flake8 --diff`` 
 - [x] Fully documented

### CODE REVIEW

**Comment by TAlonglong** [comment]
Could you please have a look at this @adybbroe or @mraspaud? Or someone else?

File: trollsched/schedule.py [hunk 1]
```diff
@@ -572,12 +642,12 @@ def parse_datetime(strtime):
     return datetime.strptime(strtime, "%Y%m%d%H%M%S")
 
 
-def save_passes(allpasses, poly, output_dir):
+def save_passes(allpasses, poly, output_dir, plot_parameters, plot_title):
```
Comment (mraspaud) [inline trollsched/schedule.py]: Maybe make the two last parameters optional not to break backwards compatibility.

**Comment by mraspaud** [comment]
@TAlonglong don't you want to add tests to make sure meos files have the right format ? that way it'll be harder to break something in the future

**Comment by TAlonglong** [comment]
Thanks for the comments @mraspaud.

**Comment by TAlonglong** [comment]
I forgot about this over the easter. Can you have a look again?

**Comment by TAlonglong** [comment]
Hm, now that is a good question. 

What I think happened is that I worked with an earlier version where save_fig was where I have it. Then this was moved to another file without me updating my branch correctly.

Let me have a look. Maybe I can just delete it.

**Comment by adybbroe** [comment]
@TAlonglong and @martin Yes, we separated the drawing in order to make the Basemap->Cartopy migration easier. So save_fig and show are now in drawing.py

**Pull request merged**